app/views/self_service/firms/edit.html.erb

```diff
@@ -3,14 +3,19 @@
                         { name: @firm.registered_name }] %>
 
 <div class="l-constrained">
-  <h1 class="t-firm-name">
-    <%= @firm.registered_name %>
-  </h1>
+  <div class="l-heading-with-inline-links">
+    <h1 class="t-firm-name">
+      <%= @firm.registered_name %>
+    </h1>
+    <% if @firm.persisted? %>
```

Comment:
Things in an `edit` partial are always going to be persisted.

So this also needs to go into the create template, now.